static/js/src/advantage/contracts-api.js

```diff
@@ -36,6 +36,15 @@ export async function getRenewal(renewalID) {
   return data;
 }
 
+export async function getCustormerInfo(accountId) {
```

Comment:
typo: `getCustomerInfo` (without the `r`)

Reply:
Whoops! Good catch